addons/core/translations/resources/en-us.yaml

```diff
@@ -89,7 +89,7 @@ host-catalog:
       label: Static
```

Comment:
Types aren't like form fields and should be simple, e.g. `static: Static` as they are for other resources..  If we need type descriptions I think we need a different way to express them.

Reply:
what do you think about doing something like this? (similarly for host/host-set)
```
host-catalog: 
  help: 
     Static: This is static helper text
     Dynamic: This is static helper text
     Azure: This is static helper text
     AWS: This is static helper text
```

Reply:
Works for me.